docs/source/_static/css/custom.css

```diff
@@ -0,0 +1,83 @@
+@import url("theme.css");
```

Comment:
just curious, what is theme.css?

Reply:
I'm not sure. It was the initial line in the conda-build CSS and I copied that over for conda. I don't know if it's necessary but @msarahan may know since I think he created the original .css in conda-build.

Reply:
whoa, what?  I don't remember anything like that.  I have no idea what I'm doing with CSS.  Maybe I was copying over some other template or rearranging things.